STUDIO/org.openl.rules.webstudio/webapp/pages/modules/administration/settings/mail/mailConfig.xhtml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ui:composition xmlns:ui="http://java.sun.com/jsf/facelets">
+
+    <style>
+        #mail-config-verification-section {
+            margin-top: 0;
+            display:none
+        }
+    </style>
+    <div class="fields">
+        <section>
+            <h3>Mail configuration</h3>
+            <div>
+                <table>
+                    <tbody>
+                        <tr>
+                            <td class="titleColumn"><label>Enable mail verification:</label></td>
+                            <td><input id="mail-config-verification" type="checkbox"/></td>
+                            <td><span id="mail-config-error" class="error"/></td>
+                        </tr>
+                    </tbody>
+                </table>
+            </div>
+            <div id="mail-config-verification-section" class="fields">
+                <section>
+                    <div>
+                        <table>
+                            <tbody>
+                                <tr>
+                                    <td class="titleColumn"><label class="required">URL:</label></td>
+                                    <td><input id="mail-config-url" type="text"/></td>
+                                </tr>
+                                <tr>
+                                    <td class="titleColumn"><label class="required">Username:</label></td>
+                                    <td><input id="mail-config-username" type="text"/></td>
+                                </tr>
+                                <tr>
+                                    <td class="titleColumn"><label class="required">Password:</label></td>
+                                    <td><input id="mail-config-password" type="password"/></td>
+                                </tr>
+                            </tbody>
+                        </table>
+                    </div>
+                </section>
+            </div>
+        </section>
+        <input id="mail-config" type="button" value="Apply All and Restart" title="Apply All and Restart"
+               class="button-primary save-button"/>
+    </div>
+    <script>//<![CDATA[
+        $j(function() {
+            const mailConfigVerificationSection = $j("#mail-config-verification-section")

```

Comment:
Good place to try to use ``ws.form.fetchFormData`` and ``ws.form.registerSubmitFormHandler`` functions